m2l31x/Devices/M2L31/Include/system_M2L31.h

```diff
@@ -27,7 +27,7 @@ extern "C" {
 /*----------------------------------------------------------------------------
   Define clocks
  *----------------------------------------------------------------------------*/
-#define __HXT       (32000000UL)    //(12000000UL)    /*!< External Crystal Clock Frequency     */
+#define __HXT       (12000000UL)    /*!< External Crystal Clock Frequency     */
```

Comment:
did you mean to just replace the number rather than pushing the old one on the side

Reply:
Yes, 32M is the temporary crystal setting and 12M is the permanent setting & sync with proprietary BSP.

Reply:
if you have any concern and prefer to push the old one on the side, I could modify as:
#define __HXT       (12000000UL)    //(32000000UL)

Reply:
Sorry I've misread the diff and then forgot to followup -- this is fine thanks for clarifying.